### ISSUE TITLE
"Preview" does not work when replying to a comment

### ISSUE THREAD

**Issue by moopet** [forem/forem#20681]
**Describe the bug**

The "preview" button does not work when replying to a comment.
In the console we see:

    Uncaught ReferenceError: handleCommentPreview is not defined
    onclick https://dev.to/moopet/comment/2d7ei:1

**To Reproduce**

1. Open any post
2. Enter some text in a reply to a comment
3. Hit "preview" instead of "submit"

**Expected behavior**

We see some sort of preview of the markdown, either in-place or as a separate interstitial page.

**Screenshots**

**Desktop (please complete the following information):**

- OS, version: Windows 11
- Browser, version: Firefox 122.01

**Additional context**

This is a continuation of https://github.com/forem/forem/issues/19290 which was closed with only a partial fix last year.

**Comment by GwartneyDev** [comment]
@moopet  This seems to be a similar issue, but wanted to check, does it look something like this https://github.com/forem/forem/issues/20701

**Comment by moopet** [comment]
@GwartneyDev no, it's not like that - in this case you can't even get to the preview in the first place. It looks like the library used for previewing things isn't loaded unless you have a post to edit already (speculation).

**Comment by GwartneyDev** [comment]
@moopet  Ah ok that is weird, will have to see if I can test it on my side at some point. Just figured I would ask in general since the other one just seemed similar.

**Comment by Link2Twenty** [comment]
I'm unable to replicate the issue, has it been solved?

**Comment by moopet** [comment]
This is still a problem, and I still see "handleCommentPreview is not defined" in the console in Chrome and Firefox.

**Comment by morfenza** [comment]
This seems to be similar to #20389, @joaoGabriel55 opened an PR to solve it! #20394
